.github/workflows/_build.yml

```
@@ -0,0 +1,206 @@
+      # the repo-root Build/ directory the same way it does when MSBuild loads
+      # the .sln.
+      # SysTool/XMLWrapper are pulled in via <ProjectReference> from the
+      # SimModelNative vcxproj, so building the latter recursively builds them.
+      - name: Build native (Release)
+        run: msbuild src\OSPSuite.SimModelNative\OSPSuite.SimModelNative.vcxproj /p:Configuration=Release /p:Platform=x64 /p:SolutionDir=${{ github.workspace }}\
+
+      - name: Build C++ tests (Release)
+        run: msbuild tests\OSPSuite.SimModelNative.Tests\OSPSuite.SimModelNative.Tests.vcxproj /p:Configuration=Release /p:Platform=x64 /p:SolutionDir=${{ github.workspace }}\
+
+      - name: Run C++ tests
+        run: Build\Release\x64\OSPSuite.SimModelNative.Tests.exe --gtest_output=html:Build\Release\x64\testLogCpp_Windows.html
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1001:info:1:14: This \x will be a regular 'x' in this context

(shellcheck)